test/wdio/css-variables/cmp.test.tsx

```diff
@@ -0,0 +1,156 @@
+import './variables.css';
```

Comment:
These tests didn't exist prior to this port, so I tried to write tests that captured the spirit of the original. Leaving some notes/comments/questions inline